axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/Synchronizer.java

```diff
@@ -85,7 +85,11 @@ private void syncAndCloseFile() {
         try {
             WritePosition toSync = syncAndCloseFile.pollFirst();
             if (toSync != null) {
-                closeFile(toSync);
+                try {
+                    closeFile(toSync);
+                } catch (Exception ex) {
+                    syncAndCloseFile.add(toSync);
```

Comment:
Should we log `ex`?

Reply:
Good catch, looked at the exception handling again and the exception code was never reached as closeFile already caught the exception. updated the error handling to make this the only place where exceptions are caught.